### PR TITLE
NEWS: update v3.1.6

### PR DESCRIPTION
**Pull request by gyuho**
For tomorrow release

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,9 @@
+etcd v3.1.6 (2017-04-19)
+- remove auth check in Status API
+- fix 'etcdctl endpoint status' crash with auth enabled
```
Comment (heyitsanthony) [inline NEWS]: can probably leave this one out since segfault is clearly broken behavior

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- fill in Auth API response header
+  - fix crash in 'etcdctl --write-out=fields'
```
Comment (heyitsanthony) [inline NEWS]: same here about the crashing, fill-in seems OK to list

**Comment by fanminshi** [comment]
lgtm

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**